certbot/tests/util.py

```diff
@@ -24,6 +24,7 @@
 from certbot import util
 
 from certbot.display import util as display_util
+from certbot.configuration import NamespaceConfig
```

Comment:
nit: Our [style guide](https://certbot.eff.org/docs/contributing.html#coding-style) recommends [only importing packages/modules](https://google.github.io/styleguide/pyguide.html#Imports) so it'd be better to just import `configuration` and use ` configuration.NamespaceConfig`.